### PR TITLE
Example project docs

### PR DESCRIPTION
**Pull request by marcinc**
This is just a sample documentation. Some projects won't require most of the docs in the sample. This is to show the format in which documentation should be produced to make it consistent across different projects.

Install [`mkdocs`](https://www.mkdocs.org/#installation)

To serve generated version of documentation with search option:
`mkdocs serve` - this will start the local server on `http://127.0.0.1:8000`.

To generate static version of the website to be published externally:
`mkdocs build --clean` - this will compile docs into a static html format (`/site` dir)

To publish as Github pages (off master):
`mkdocs gh-deploy` - this will publish compiled documentation to *public* Github pages for that repo. NOTE: this will publish to PUBLICLY accessible Github pages even if the repo itself is private so use with caution!

### CODE REVIEW

**Review by KashifSaadat — APPROVED** [review]
LG

File: docs/backup_and_restore.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+# Restore
+
+Describe project data restore strategy and process. 
+
+## On-demand restore
+
+Describe restore procedure.
+
+## Distaster recovery procedure
```
Comment (KashifSaadat) [inline docs/backup_and_restore.md]: `Distaster` => `Disaster`

File: docs/backup_and_restore.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+Describe disaster recovery procedue in case of total outage.
```
Comment (KashifSaadat) [inline docs/backup_and_restore.md]: `procedue` => `procedure`

File: docs/installation.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+Provide instructions on how to install / setup this project.
+
+## Download source
+
+```
+# Instructions here...
+```
+
+## Build
```
Comment (KashifSaadat) [inline docs/installation.md:11]: Does this overlap in some respect with `docs/build.md`?

**Review by KashifSaadat — CHANGES_REQUESTED** [review]

File: docs/installation.md (hunk 3 above)
Comment (marcinc) [reply docs/installation.md:11]: Just an example really... Could potentially skip those steps and leave it person writing docs.

File: docs/backup_and_restore.md (hunk 2 above)
Comment (marcinc) [reply docs/backup_and_restore.md]: 🎱

File: docs/backup_and_restore.md (hunk 1 above)
Comment (marcinc) [reply docs/backup_and_restore.md]: 🎱

**Review by KashifSaadat — APPROVED** [review]
LGTM

**Pull request merged**